app/javascript/crayons/formElements/FormField/index.js

```diff
@@ -0,0 +1 @@
+export * from './FormField';
```

Comment:
I noticed how CodeClimate complains there's no "named export", should there be? IIRC it's quite a recent feature: https://codeclimate.com/github/thepracticaldev/dev.to/pull/7112

Reply:
This is a false negative. This is common practice in frontend.

Reply:
I've fixed the eslint config in codeclimate for this.